CONTRIBUTING.md

```diff
@@ -118,9 +126,9 @@ Despite being named 'unit', these are integration tests, making sure public API
 ```
 npm run unit
 ```
-- To filter tests by name:
+- To run tests in parallel, use `-j` flag:
 ```
-npm run unit -- --filter=waitFor
```

Comment:
Can we still filter?

Reply:
We can't. Though it's trivial to implement if we ever need it back.